apps/play/src/style.css

```
@@ -652,6 +652,39 @@ piano-keyboard.visible {
+  align-items: center;
+  justify-content: center;
+  gap: 0.25rem;
+  width: 100px;
+  overflow-x: visible;
+
+  /* > .input-source-select {
+    max-width: 50px;
+  } */
+  /* > .input-device-select {
+    outline: lime 2px solid;
+  }*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)